testing/tests/DevExpress.ui/defaultOptions.tests.js

```diff
@@ -665,17 +664,6 @@ testComponentDefaults(Toast,
     }
 );
 
-testComponentDefaults(Toolbar,
```

Comment:
Should we rework this behavior dependent on `platform == ios`?
/cc @jsdmitry

Reply:
We have planned to discuss this refactoring

Reply:
We are going to get rid of the 'actionSheet' strategy and the sybmenuType private option because we don't need. Using this private option into our code is not obvious to users. The 'ios' platform or theme should not affect this.

We are going to do this as part of a separate task when we will have free resources for this. We would't plan this work in the 20.2 release timeframe.